### PR TITLE
Unify Export Options

### PR DESCRIPTION
**Pull request by barreeeiroo**

### CODE REVIEW

**Comment by barreeeiroo** [comment]
@ewpatton This is ready for review now. It contains the changes in the Dialog layout. I've also added some minor changes, like remove gitignore duplicates, force download progress to start at 0% always, etc.

File: appinventor/appengine/src/com/google/appinventor/client/explorer/commands/ShowBarcodeCommand.java [hunk 1]
```diff
@@ -54,54 +59,105 @@ public boolean willCallExecuteNextCommand() {
   public void execute(final ProjectNode node) {
     // Display a barcode for an url pointing at our server's download servlet
     String barcodeUrl = GWT.getHostPageBaseURL()
-      + "b/" + Ode.getInstance().getNonce();
+            + "b/" + Ode.getInstance().getNonce();
```
[the remark this answers is not in the document]
Comment (barreeeiroo) [reply appinventor/appengine/src/com/google/appinventor/client/explorer/commands/ShowBarcodeCommand.java:62]: Sure, I just changed the IntelliJ config to it 👍

**Comment by barreeeiroo** [comment]
> Also, when I tested this locally I still saw the progress bar start at 50% and then jump back to 30%, so I'm not sure that is affected by your changes in the buildserver module. Can you confirm you see the correct behavior?

Yeah, you are right 😅 
I thought it was fixed, but after running some tests it appeared again.

Anyway, I will investigate it further with the Compiler refactor, it's not something important now

**Comment by ewpatton** [comment]
@barreeeiroo Agreed.

**Comment by barreeeiroo** [comment]
I just realized I've committed the review changes into the next PR (#3). Just to cross-reference them, the commits that solve the review changes are:

- Fix continuation indentation [a08fc5a]
- Rename method name to include "Title" [455c88b]
- Change OK to Dismiss button [8ebb6ab]
- Removed old code [f0b9538]

**Pull request merged**